ansible/roles/django/templates/settings_secret.py.j2

```diff
@@ -0,0 +1,16 @@
+from requests.auth import HTTPBasicAuth
+
+SECRET_KEY = '{{ secret.secret_key }}'
+KRATOS_ADMIN_PWD = '{{ secret.couch_pwd }}'
+
+DATA_DIR = '{{ data_dir }}/django'
+
+SOCIAL_AUTH_GITHUB_ENTERPRISE_KEY = '{{ github_enterprise.OAUTH.KEY }}'
```

Comment:
I'm curious about the decision to templatize the secret settings, rather than just expecting the entire file to be provided by the pantheon-config source.  This is enforcing a rather rigid config structure, what if the end user had github enterprise, but not public github, or vice versa? What if the end user doesn't need github.WELCOME_TEAM?